### PR TITLE
Add person

### PR DESCRIPTION
**Pull request by anitaschiller**
Branch to user story #1 

In order to be able to add a new person, I added a form.
After clicking on the submit button, the new person including his/her christmas ideas should be added to the `personsToSendAGift` useState.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -10,12 +8,26 @@ function App() {
...
   return (
     <div className="App">
       <h1>Weihnachts-Ideen</h1>
-      {personsToSendAGift.map((person, index) => (
-        <Card key={index} personToBeGifted={person} />
-      ))}
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="newPerson">Add a new person</label>
+        <input type="text" id="newPerson" name="newPerson" />
+        <label htmlFor="newIdeas">Add your ideas</label>
```
Comment (ThomasKirst) [inline src/App.jsx]: It would be great if the user gets a note on how to enter the ideas (comma-separated)

File: src/App.jsx [hunk 2]
```diff
@@ -10,12 +8,26 @@ function App() {
...
   const [personsToSendAGift, setPersonsToSendAGift] = useState(initialPersons);
 
+  function handleSubmit(event) {
+    event.preventDefault();
+    const newPerson = event.target.newPerson.value;
+    const newIdeasString = event.target.newIdeas.value;
+    const newIdeasArray = newIdeasString.split(', ');
+    const newPersonObject = { name: newPerson, ideas: newIdeasArray };
+    setPersonsToSendAGift([...personsToSendAGift, newPersonObject]);
+    console.log(personsToSendAGift);
```
Comment (ThomasKirst) [inline src/App.jsx]: Please remove `console.log` ;)

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Looks overall good to me!

It would be nice to have a README.md telling the dev how to set up the project and run the app.

Apart from that: great work! :)

**Comment by anitaschiller** [comment]
Hi Thomas, thank you for your review and comments. Please check once again, thx.

**Comment by ThomasKirst** [comment]
Awesome! Nice form, I like :)

**Review by ThomasKirst — APPROVED** [review]
Great, go for deploy!

**Pull request merged**